doc/manuals/admin/extra/ha.md

```diff
@@ -0,0 +1,441 @@
+# Create an Orion Context Broker cluster in High Availability
+
+This documentation describe how to set-up on a generic infrastructure an
+Orion Context Broker cluster in High Availability.
+
+The architecture is made by three logical layers:
+* A Load Balancer (implemented using HA Proxy)
+* The Orion Context Broker API
+* The MongoDB ReplicaSet backend
+
+The ideal solution for providing Active-Active access to the HA Proxy cluster
+(and consequently of the Context Broker) is the adoption of a DNS service that
+support multiple IPs for a single name.
+This will allow the clients to use a round-robin solution to pick the actual 
+host to connect with, in case of failure, the second will be used and so on.
+The alternative is the usage of a VIP mechanism to provide Active-Passive access
+to the HA Proxy cluster. In this case, the active instance of HA proxy will be
+only one, in case of a failure of the active HA Proxy, one of the other HA Proxy
+will take on the Virtual IP.
+
+The picture below present the network architecture of the deployed cluster.
+
+```
+    +------------+    +------------+    +------------+
```

Comment:
ASCII art is great :) but I think this should be replaced by an actual image (i.e. png file), as this is the style we have in the documentation (e.g. https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/Orion-ioflows.png used from https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/diagnosis.md#io-flows)

I can pass you by email the source ppt used for generated the image, so you can use the common style.

Reply:
yup I agree, it can be clearer showing a regular picture